notebooks/ingest_newville_example.ipynb

```diff
@@ -34,10 +34,47 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "id": "1ff6333b-690f-4119-a2b2-bbe5c94c3112",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
All of the notebooks we store in GitHub should have their outputs stripped. This is because often outputs can inadvertently contain images or just lots of text, and this can be very space-intensive. There are a variety of ways to do this, including by using the command palette in Jupyter or by doing it from the command line, e.g. [this](https://stackoverflow.com/questions/28908319/how-to-clear-jupyter-notebooks-output-in-all-cells-from-the-linux-terminal).